NAMESPACE

```diff
@@ -2,4 +2,12 @@
 
 export(analyse_oneshot_spatial)
 export(analyse_rasters_spatial)
+export(basedata)
+export(install_perlbrewr)
+export(load_data_)
```

Comment:
Not all of these need to be exported.  

Utility funcs like ```install_perlbrewr``` can be exported but any data analysis needs to go via the R6 basedata object.

Reply:
Which ones would you like to export? :) Do you want to change that from your end by removing the roxygentag @export

Reply:
If you tell me which ones I can also change :)

Reply:
```basedata``` and ```install_perlbrewr``` might be all we need from the set being added in this PR.  

The ```run_*``` functions all require book-keeping that the ```basedata``` object handles.  ```process_tabular_results``` is there to convert data from JSON to tabular form.